plugin-server/tests/main/ingestion-queues/session-recording/replay-events-ingester.test.ts

```diff
@@ -0,0 +1,148 @@
+import { captureException, captureMessage } from '@sentry/node'
+import { DateTime } from 'luxon'
+import { mkdirSync } from 'node:fs'
+import path from 'path'
+
+import { defaultConfig } from '../../../../src/config/config'
+import { OffsetHighWaterMarker } from '../../../../src/main/ingestion-queues/session-recording/services/offset-high-water-marker'
+import { ReplayEventsIngester } from '../../../../src/main/ingestion-queues/session-recording/services/replay-events-ingester'
+import { Hub, PluginsServerConfig, TimestampFormat } from '../../../../src/types'
+import { createHub } from '../../../../src/utils/db/hub'
+import { castTimestampOrNow } from '../../../../src/utils/utils'
+import { createIncomingRecordingMessage } from './fixtures'
+
+jest.mock('@sentry/node', () => ({
+    captureException: jest.fn(),
+    captureMessage: jest.fn(),
+}))
+
+describe('replayEventsIngester', () => {
+    const config: PluginsServerConfig = {
+        ...defaultConfig,
+        SESSION_RECORDING_LOCAL_DIRECTORY: '.tmp/test-session-recordings',
+    }
+
+    let ingester: ReplayEventsIngester
+
+    let hub: Hub
+    let closeHub: () => Promise<void>
+
+    beforeAll(() => {
+        jest.useFakeTimers({
+            // magic is for evil wizards
+            // setInterval in blob consumer doesn't fire
+            // if legacyFakeTimers is false
+            // 🤷
+            legacyFakeTimers: true,
+        })
+        mkdirSync(path.join(config.SESSION_RECORDING_LOCAL_DIRECTORY, 'session-buffer-files'), { recursive: true })
+    })
+
+    beforeEach(async () => {
+        ;[hub, closeHub] = await createHub()
```

Comment:
`const`?

Reply:
That redeclares them rather than assigns them
(I did copy paste this from another file without reading it to be fair 🤣)

We declare hub and closeHub up above and then destructure into those variables on this line.